src/app.html

```diff
@@ -48,6 +48,14 @@
 
 		<link rel="manifest" href="/manifest.json" crossorigin="anonymous" />
 		<script defer data-domain="waterneuron.fi" src="https://plausible.io/js/script.js"></script>
+		<script type="application/ld+json">
+			{
+				"@context": "https://schema.org",
+				"@type": "Organization",
+				"url": "https://waterneuron.fi",
+				"logo": "https://waterneuron.fi/favicons/favicon.ico"
+			}
+		</script>
```

Comment:
Where does that come from?

Reply:
It's because of the problem we have with our SEO (the logo doesn't appear on a browser search). It might be the solution